src/sortMethods.js

```diff
@@ -0,0 +1,107 @@
+
+const sortMethods = {
+
+        SELECT : {
```

Comment:
I think it's ok to get rid of this and just choose `TITLE_ASCENDING` as the default sort method, as users will expect that behaviour.

Reply:
The problem: the dropdown 'selects' the top option by default so clicking on it does not fire an onChange and becomes basically a 'dumb' button until another button is selected. A dummy top option allows you to, at all times, select and sort by all options. Alternative is to have a seperate Submit button. 

Also having 'Select from one of the below:' at the top prompts user action.